### PR TITLE
Remove AUTOGENERATED docs

### PR DESCRIPTION
**Pull request by jonreding2010**
Deleted all the Autogenerated files in MAQS_5.
Updated sidebar.md

### CODE REVIEW

File: docs/MAQS_5/_sidebar.md [hunk 1]
```diff
@@ -32,54 +34,21 @@
 		- [xii. Kill Driver](MAQS_5/SeleniumUtils.md?id=KillDriver)
 		- [xiii. FAQ](MAQS_5/SeleniumFAQ.md)
 
-	- [iv. Appium](MAQS_5/Appium_AUTOGENERATED/LandingPage.md)
```
Comment (FermJacob) [inline docs/MAQS_5/_sidebar.md]: From our discussion, we need to add basic Appium and email documentation in

Comment (TroyWalshProf) [reply docs/MAQS_5/_sidebar.md]: And base test

Comment (jonreding2010) [reply docs/MAQS_5/_sidebar.md]: Base, Appium, and Email documentation has been added to the _sidebar. md

**Comment by jonreding2010** [comment]
Full overhaul of the Documentation organization
Added Documentation pages for all classes.

**Pull request merged**